### ISSUE TITLE
ModuleNotFoundError for defusedxml in anaconda console

### ISSUE THREAD

**Issue by louloup22** [conda-forge/zeep-feedstock#8]
I just followed the steps in the Readme to install zeep with the channel conda-forge. It worked and also installed other packages at the same time (such as defusedxml 0.5.0)
However when running my script with anaconda3, I get such error: 
```
 File "~/anaconda3/lib/python3.6/site-packages/zeep/loader.py", line 3, in <module>
    from defusedxml.lxml import fromstring

ModuleNotFoundError: No module named 'defusedxml'
```

When I install zeep with python3 pip install and run my script in python3 console, I do not get such error (not same install). 

Any idea why I get such errors ? 

Thanks.

**Comment by ghost** [comment]
Hi @louloup22

You can check if the package is installed using the command 
`conda list`
Did you try updating or cleaning the package?

```
conda clean
conda update
```

More Importantly, the version available for Zeep via conda as of now is only 2.5.0, and I highly recommend using 3.1.0, from pip, as the documentation has also been updated.
Check out the release notes:
[https://python-zeep.readthedocs.io/en/master/changes.html]()

Looking at the previous pull requests, it seems there was an attempt to upgrade but failed.
I hope someone updates this soon. 

Thanks.

**Comment by Cadair** [comment]
This has now been updated. Let me know if you are still having issues.
